src/routes/[...slugs=myanimelist]/episode/[id=int]/+page.svelte

```diff
@@ -2,8 +2,7 @@
     import { page } from "$app/stores";
     import EpisodePage from "$components/pages/episode/index.svelte";
 
-    let episode_number: number;
-    $: episode_number = Number($page.params.id);
+    let episode_number = Number($page.params.id);
```

Comment:
Okay it works when we manually refresh the page.
But we need to change it on each button click, url changes but this not :|